app/i18n/locales/nl.json

```diff
@@ -445,6 +455,7 @@
   "Welcome": "Welkom",
   "Whats_your_2fa": "Wat is je 2FA code?",
   "Without_Servers": "Zonder Servers",
+  "Workspaces": "Werkplekken",
```

Comment:
"Werkruimtes" (a bit difficult to guess what the correct word is without context)